backend/src/main/java/lt/vu/tube/util/AWSS3Utils.java

```diff
@@ -155,4 +166,34 @@ public void setDeleteObjectResponse(DeleteObjectResponse deleteObjectResponse) {
         }
     }
 
+    private static class MultipartUploadPartContainer {
```

Comment:
We have several classes in one file?

Reply:
As it is a class only used in this class, an inner class is suitable for this